packages/components/pie-textarea/test/visual/pie-textarea.spec.ts

```diff
@@ -85,6 +85,30 @@ test.describe('disabled', () => {
     });
 });
 
+test.describe('placeholder', () => {
+    test('should render correctly when `placeholder` is passed', async ({ page, mount }) => {
```

Comment:
Can we get a couple more test cases, please:
1. Disabled with placeholder
2. With placeholder and value (placeholder should not be visible)
3. Readonly with value (maybe this can replace the empty readonly test below)